SeaPublicWebsite.Data/Migrations/SeaDbContextModelSnapshot.cs

```diff
@@ -38,7 +38,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.HasKey("Id");
 
-                    b.ToTable("DataProtectionKeys");
+                    b.ToTable("DataProtectionKeys", (string)null);

```

Comment:
Why is there no change here removing the `ReturningUser` column? It should have been part of the snapshot file before removing the migration

Reply:
I took the original file from the main branch and then created the necessary migration. Maybe that is the reason?